### PR TITLE
Update input processing

### PR DESCRIPTION
**Pull request by brianloyal**
Two updates:

1. Only look at .fasta or .fa files when running pretrained inference. This will help in cases where the fasta files aren't in their own folder and will help use use the same infrastructure to handle AlphaFold jobs.
2. Change the approach for extracting tags from fasta files to account for cases where sequences span multiple lines (As in the case of files generated by biopython)

Feel free to squash the commits

### CODE REVIEW

File: run_pretrained_openfold.py [hunk 1]
```diff
@@ -230,18 +230,18 @@ def main(args):
     prediction_dir = os.path.join(args.output_dir, "predictions")
     os.makedirs(prediction_dir, exist_ok=True)
 
-    for fasta_file in os.listdir(args.fasta_dir):
+    for fasta_file in [f for f in os.listdir(args.fasta_dir) if os.path.splitext(f)[1] in [".fasta", ".fa"]]:
```
Comment (decarboxy) [inline run_pretrained_openfold.py:233]: Purely from a readability standpoint it might be worth extracting this list comprehension to a short function

**Review by decarboxy — COMMENTED** [review]
I like it!

File: run_pretrained_openfold.py (hunk 1 above)
Comment (brianloyal) [reply run_pretrained_openfold.py:233]: Works for me!

**Comment by gahdritz** [comment]
LGTM. @brianloyal would you mind resolving conflicts?

**Comment by brianloyal** [comment]
Resolved conflicts and updated PR. 

Turns out that the multi-line sequence problem I saw only happens when you have a description in your fasta header with another ">" symbol. This is bad form but, funny story, if you don't provide a description to the Biopython `Bio.SeqRecord` object `<unknown description>' and screws up the format for OpenFold. I added a quick fix to the `parse_fasta` method rather than my older approach.

I also split out the one-liner to only pull in .fasta or .fa files into it's own function for readability.

**Pull request merged**